avocado/plugins/yaml_to_mux.py

```diff
@@ -18,6 +18,7 @@
 import logging
 import os
 import re
+import sha
```

Comment:
The ordering notice is quite important. What it means is, that:

```yaml
first:
    foo: bar
second:
    bar: baz
```

```yaml
second:
    bar: baz
first:
    foo: bar
```

Produces the same variant-id. Without the manual sorting it'd produce different ids because the multiplexer is ordered (therefor the order of `first` and `second` matters inside the variant, even though the final params are the same)

The default_params has no relation to the variant-id whatsoever.